Patch-mainline: yes

Some versions of the IBM 2104-DU3 disk enclosure
have been observed to hang Inquiries to non zero
LUNs to the SES device. This device only has LUN 0,
so this patch adds it to the BLIST to prevent scsi
core from scanning beyond LUN 0.

Signed-off-by: Brian King <brking@us.ibm.com>

---

 linux-2.6-bjking1/drivers/scsi/scsi_devinfo.c |    2 ++
 1 files changed, 2 insertions(+)

diff -puN drivers/scsi/scsi_devinfo.c~scsi_2104-DU3_blist drivers/scsi/scsi_devinfo.c
--- linux-2.6/drivers/scsi/scsi_devinfo.c~scsi_2104-DU3_blist	2006-04-25 14:52:21.000000000 -0500
+++ linux-2.6-bjking1/drivers/scsi/scsi_devinfo.c	2006-04-25 14:55:33.000000000 -0500
@@ -56,6 +56,8 @@ static struct {
 	{"DENON", "DRD-25X", "V", BLIST_NOLUN},			/* locks up */
 	{"HITACHI", "DK312C", "CM81", BLIST_NOLUN},	/* responds to all lun */
 	{"HITACHI", "DK314C", "CR21", BLIST_NOLUN},	/* responds to all lun */
+	{"IBM", "2104-DU3", NULL, BLIST_NOLUN},		/* locks up */
+	{"IBM", "2104-TU3", NULL, BLIST_NOLUN},		/* locks up */
 	{"IMS", "CDD521/10", "2.06", BLIST_NOLUN},	/* locks up */
 	{"MAXTOR", "XT-3280", "PR02", BLIST_NOLUN},	/* locks up */
 	{"MAXTOR", "XT-4380S", "B3C", BLIST_NOLUN},	/* locks up */
_
